### DISCUSSION TITLE
Anyone else couldn't use this chrome extension?

### DISCUSSION THREAD

**Discussion by crazywiden** [richards199999/Thinking-Claude#40] [General]
I installed this chrome extension and enabled it
![image](https://github.com/user-attachments/assets/945a221b-f091-49c5-a593-cfa6b173c330)

but when I am chatting with Claude. I still couldn't get the thinking process. 
![image](https://github.com/user-attachments/assets/ada5e92f-9f97-4e9f-810a-19cc2c749fcb)

I am using Claude premium and I have no idea where should I start debugging this. Any suggestions of how to debug is appreciated!

**Comment by lumpinif** [comment]
It looks like you didn't use the prompt first. You can't make Claude think if you don't use these [prompts](https://github.com/richards199999/Thinking-Claude/tree/main/model_instructions) first. If Claude doesn't think you won't see the thinking process no matter how.

**Reply by crazywiden** [reply]
Thanks @lumpinif for the reply! Originally I just thought the extension will do the prompts magic for me. Now I can get it work!
![image](https://github.com/user-attachments/assets/aaf1b801-46b2-4d76-8f44-0b9875aba484)

Appreciate it!

**Reply by lumpinif** [reply]
you're welcome! however from what I can see from screenshots the extension is not loaded right please refresh the page to let it load which will give you a better experience for reading the thinking process.

btw, your intuition is correct! I will be working on something like how you just thought, which is the magic feature and better experience.

**Comment by yeyuewubian** [comment]
你好，我也遇到了这个问题，我没有订阅pro版本，chrome也安装了think claude，但是在Claude里无法唤醒think来使用，请问我怎么办？
chrome也是最新版本，mac os  Monterey 12.7.6版本的系统
![1736397513262](https://github.com/user-attachments/assets/8f084194-5ce2-452c-8d42-b70959d74164)
![WX20250109-123850@2x](https://github.com/user-attachments/assets/0adb2e59-f16b-4847-8585-c63d6bd5bb2c)
![WX20250109-124002@2x](https://github.com/user-attachments/assets/cbd80dcc-3330-45e9-b858-30be79c9a384)

**Reply by crazywiden** [reply]
![image](https://github.com/user-attachments/assets/ad747b37-fc5c-4d62-9b10-0f9e5a261eb6)
你需要自己新建一个project，然后把 这个里面的system prompt输进去
https://github.com/richards199999/Thinking-Claude/blob/main/model_instructions/v5.1-extensive-20241201.md

之后这个插件才能起作用

**Reply by lumpinif** [reply]
看起来你安装了两个插件版本，并同时启动了，尝试卸载或关闭老版本插件

**Reply by yeyuewubian** [reply]
两个插件版本都卸载了，然后升级Chrome「版本 132.0.6834.111（正式版本） (arm64)」，安装了Chrome应用商店里的3.1.2版本；依然无法使用。
尝试按照提示再Claude对话框里“创建和编辑样式”，粘贴了[https://github.com/richards199999/Thinking-Claude/blob/main/model_instructions/v5.1-extective-20241201.md](https://github.com/richards199999/Thinking-Claude/blob/main/model_instructions/v5.1-extensive-20241201.md)，然后Cloude崩了
<img width="632" alt="image" src="https://github.com/user-attachments/assets/d7199a34-ca9c-40f8-97a3-7d200fe0178d" />

至此，我想问，Thinking 是不是只能应用于Cloude pro版本？免费的版本不能用啊？

**Reply by yeyuewubian** [reply]
这个翻译有点让人头疼，这段有两个翻译错误，第一个，两个版本都卸载了，并且重新安装了3.1.2版本；第二个，您的这个扩展程序是不是只能在Cloude pro版本里用？

**Reply by lumpinif** [reply]
![image](https://github.com/user-attachments/assets/af902d54-7e44-4f8e-bb03-e77027083de7)

请安装最新版，然后参考首页README的教程和演示视频来操作，我刚刚测试过一切正常的，我测试的也是免费版，这个和pro没关系。

**Reply by lumpinif** [reply]
如果你注意看首页介绍的第一个视频，你也会发现我在Demo中使用的就是免费版Claude

**Comment by lumpinif** [comment]
just tested it again with chrome v3.2.2 extension:
![Screenshot 2025-01-09 155445](https://github.com/user-attachments/assets/c8eda922-5ca2-4cd5-87fd-1301d607c552)

![Screenshot 2025-01-09 160344](https://github.com/user-attachments/assets/abe47b22-0a0a-440e-bc03-03dddbcd38f0)

make sure you update the extension to the latest or manually downloaded the latest extension and install it
